.github/workflows/auto-update-gh-pages.yml

```diff
@@ -16,17 +16,37 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      
-      # Generate Doxygen output files in ./docs directory using Doxyfile in base directory
+
+      # Generate Doxygen output files in ./doc/content/html directory using Doxyfile in
+      # base directory
       - name: Doxygen Action
         uses: mattnotmitt/doxygen-action@v1
         with:
           working-directory: .
       
+      # Install MOOSE Conda environment & update MooseDocs documentation
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          mamba-version: "*"
+          channels: conda-forge,https://conda.software.inl.gov/public
+          activate-environment: moose-env
+      - shell: bash -l {0}
+        run: |
+          conda deactivate
+          conda activate moose-env
+          mamba install moose-tools=2021.07.14 moose-libmesh=2021.07.14
+          conda deactivate
+          conda activate moose-env
+          git submodule init && git submodule update
+          make -j 2
+          cd doc
+          ./moosedocs.py build --destination html
+      
       # Merge Doxygen output files in ./docs into gh-pages branch base directory
       - name: GitHub Pages action
         uses: peaceiris/actions-gh-pages@v3
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          publish_dir: ./docs
+          publish_dir: ./doc/html
```

Comment:
Should this be `./doc/content/html`? Your comment on line 20 of this workflow says so.

Reply:
Good catch. I was switching back and forth between the two options. The comment on line 20 is wrong.

Reply:
Actually I just realized both are correct. Doxygen builds into `./doc/content/html`. MooseDocs then parses the Doxygen html files in `./doc/content/html` and outputs the full website html files into `./doc/html`.